skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt

```diff
@@ -1,6 +1,10 @@
 package org.jetbrains.skiko
 
-actual val hostOs: OS = OS.JS
+import kotlinx.browser.window
+
+actual val hostOs: OS by lazy {
```

Comment:
Why should we store (by lazy) the value?

Reply:
i thought about `get() =`, but all variants will work here

Reply:
I decided to not add a getter to make sure the value is calculated only once. 
The value can potentially change when the app runs (by a client), but it's not a valid case to support. We (in compose) expect this value never changes.